rdrf/rdrf/testing/unit/tests.py

```diff
@@ -1573,10 +1573,17 @@ class Time:
         assert result == "testuser\n", "Expected testuser instead got [%s]" % result
 
         # parents are detected
-        self._setup_user("testuser", Time.LONG_AGO, group="parents")
-        result = self._run_command(registry_code="foobar", days=365)
-        assert result == "testuser\n", "Expected testuser instead got [%s]" % result
-
+        parent_feature = True
+        # Check parent feature is enabled.
+        try:
+            __import__('angelman.parent_view')
```

Comment:
As a stopgap solution this is ok, but we should try to eliminate having to do this in future by ultimately merging angelman into base rdrf